test/unit/matchers/matchers_test.rb

```diff
@@ -0,0 +1,118 @@
+require "minitest/autorun"
+require "rspec/matchers"
+require "rspec/expectations"
+require "matchers/matchers"
+
+require "pry"
+
+describe 'inspec matchers' do
+  describe 'cmp matcher' do
+    include RSpec::Matchers
+
+    def rspec_expect value, &block
+      ::RSpec::Expectations::ExpectationTarget.for(value, block)
+    end
+
+    def assert_cmp exp, act
```

Comment:
I think `exp` means `expectation`, but I don't know what `act` refers to. Could you use more expressive variable names?

Reply:
"actual". I use this all over (esp in the doco for minitest) so I make assumptions about it's "obviousness". What I really want is a better spoken language that has same-length antonyms (true vs false, local vs remote, etc).

Reply:
fixed